src/utils/wrapSVGText.ts

```diff
@@ -0,0 +1,53 @@
+export default (textElement: SVGTextElement, text: string, width: number) => {
```

Comment:
Rather than `utils/wrapSVGText.ts`, why not have everything in `WrapLabel`? ie `WrapLabel/index.tsx` & `WrapLabel/wrapSVGText.ts`

Reply:
Actually...